benefits/eligibility/views.py

```diff
@@ -53,8 +53,8 @@ def index(request, agency=None):
             flow = EnrollmentFlow.objects.get(id=flow_id)
             session.update(request, flow=flow)
 
-            types_to_verify = agency.type_names_to_verify(flow)
-            analytics.selected_verifier(request, types_to_verify)
+            type_to_verify = flow.system_name
+            analytics.selected_verifier(request, type_to_verify)
```

Comment:
Let's get rid of this intermediate variable entirely. No need for it. Just use `flow.system_name` directly.